docs: add more details on critical inlining. (#64292)
angular/angular: adev/src/content/tools/cli/build.md

Angular supports [Tailwind CSS](https://tailwindcss.com/), a utility-first CSS framework.

To integrate Tailwind CSS with Angular CLI, see [Using Tailwind CSS with Angular](guide/tailwind)

## Critical CSS inlining

Angular can inline the critical CSS definitions of your application to improve [First Contentful Paint (FCP)](https://web.dev/first-contentful-paint).

> This option is enabled default and can be desibled in the [styles customization options](reference/configs/workspace-config#styles-optimization-options).

Reviewer: ```suggestion
This option is enabled default. You can disable this inlining in the [`styles` customization options](reference/configs/workspace-config#styles-optimization-options).
```
